ee/tabby-webserver/src/service/cron/job.rs

```diff
@@ -0,0 +1,44 @@
+use job_scheduler::Job;
+use tracing::error;
+
+use crate::{path::job_runs_dir, schema::SchedulerRunMeta};
+
+pub(crate) fn repository_job<'a>(job_name: &'a str, schedule: &'a str) -> Job<'a> {
+    let job_name = job_name.to_owned();
+
+    Job::new(schedule.parse().unwrap(), move || {
+        let start_time = chrono::Utc::now();
+
+        let exe = std::env::current_exe().unwrap();
+        let mut cmd = std::process::Command::new(exe);
```

Comment:
you could use https://docs.rs/tokio/latest/tokio/process/index.html to make it async